src/site/xdoc/server/advantages.xml

```diff
@@ -31,13 +31,12 @@
 
         <subsection name="Standard">
 
-          <p>100% pure Java Mail server designed to be a complete and portable enterprise
-            mail engine solution.</p>
+          <p>100% pure Java Mail server designed to be a complete enterprise mail engine solution.</p>
 
-          <p>Supports currently available IETF protocols, including SMTP, LMTP, POP3, IMAP4, ManageSIEVE. We also plan to support JMAP.</p>
+          <p>Supports currently available IETF protocols, including SMTP, LMTP, POP3, IMAP4. We also plan to support JMAP. ManageSIEVE support is experimental.</p>
 
           <p>Apache James Server is able to store user and message data either in any JPA-compatible database,
-            in Cassandra (allowing fast, reliable, even real-time replicated storage) and on a file-system (Maildir).</p>
+            in Cassandra (allowing fast, reliable, even real-time replicated storage) in combinaison with an objectstore.</p>
 
```

Comment:
Might as well mention the type of object store here, so people have a better idea what to expect. How about: "optionally in combination with an S3 compatible object store."

Reply:
Yes for this one agree.

We should be careful IMO of not giving too much technical details inside very generic documentation page that might apply to other James packagings as well...